doc/0000-ccl_note/main.tex

```diff
@@ -550,162 +448,213 @@ \subsubsection{Cosmic emulator}
+  \item and $k_s$ [$h/$Mpc]: the wavenumber that determines the scale of the stellar profile
 \end{itemize}
 
-If the BCM parameters are not specified and the user sets \ccl to compute the power spectrum with baryonic feedback included, \ccl will assume the default parameters of \citet{Schneider15}. The user may pass these explicitly if desired.
-
-Other baryonic impact models might be incorporated in the future as they become available.
+If the BCM parameters are not specified and the user sets \ccl to compute the power
+spectrum with baryonic feedback included, \ccl will assume the default parameters
+of \citet{Schneider15}.
 
 \subsubsection{Modified gravity ($\mu, \Sigma$)}
-\ccl supports the quasistatic parameterization of modified gravity with scale-independent deviations from GR which arise at late times (currently with functional form given in equation \ref{muSigform}). Under these conditions, the matter power spectrum in a modified gravity scenario can be computed simply by re-scaling the GR matter power spectrum with the modified gravity growth factor.
+\ccl supports the quasistatic parameterization of modified gravity with
+scale-independent deviations from GR which arise at late times (currently with
+functional form given in equation \ref{muSigform}). Under these conditions,
+the matter power spectrum in a modified gravity scenario can be computed simply
+by re-scaling the GR matter power spectrum with the modified gravity growth factor.
+
+For each power spectrum calculation method above, we first compute the power spectrum
+in GR at a grid in $a$ and $k$ as usual, in preparation for producing splines in
+these variables. If ${\tt mu\_0} \ne 0$, we then take the ratio of the growth factor
+$D$ at the given value of ${\tt mu\_0}$ to the GR (${\tt mu\_0}=0$) growth factor,
+and multiply the GR power spectrum by this ratio squared:
 
-For each power spectrum calculation method above, we first compute the power spectrum in GR at a grid in $a$ and $k$ as usual, in preparation for producing splines in these variables. If ${\tt mu\_0} \ne 0$, we then take the ratio of the growth factor $D$ at the given value of ${\tt mu\_0}$ to the GR (${\tt mu\_0}=0$) growth factor, and multiply the GR power spectrum by this ratio squared:
 \begin{equation}
 P(k,z) = \left(\frac{D^{\mu}(z)}{D^{\rm GR}(z)}\right)^2 P^{\rm GR}(k,z).
 \label{Pkmu}
 \end{equation}
-We then set up splines for the power spectrum as usual. 
-
-Note that the $\mu, \Sigma$ quasistatic parameterization of deviations from GR is technically only valid in the linear regime and at sub-horizon scales. We allow the user to do this re-scaling for the non-linear power spectrum, but issue a warning. Computing the matter power spectrum under this paramterisation of modified gravity will not produce physically meaningful results on very large or small scales. 
+We then set up splines for the power spectrum as usual.
 
-\subsubsection{Spline parameters}
```

Comment:
Did you move this somewhere else in the repository? Or just code docs? These parameters determine the accuracy of CCL, so I think we should have them documented somewhere.